src/defineFunction.js

```diff
@@ -21,7 +22,7 @@ export type FunctionHandler<NODETYPE: NodeType> = (
     context: FunctionContext,
     args: AnyParseNode[],
     optArgs: (?AnyParseNode)[],
-) => ParseNode<NODETYPE>;
+) => UnsupportedCmdParseNode | ParseNode<NODETYPE>;
```

Comment:
I had to flip the order to make flow happy.  I'll add a link to the issue in the code.